builtin/credential/radius/path_login.go

```diff
@@ -20,6 +20,13 @@ import (
 func pathLogin(b *backend) *framework.Path {
 	return &framework.Path{
 		Pattern: "login" + framework.OptionalParamRegex("urlusername"),
+
+		DisplayAttrs: &framework.DisplayAttributes{
+			OperationPrefix: operationPrefixRadius,
+			OperationVerb:   "login",
+			OperationSuffix: "|with-username",
```

Comment:
Does this result in 2 separate operation IDs for w/ and without user name

Reply:
Yeap:
```
radius-login
radius-login-with-username
```